### PR TITLE
160482625 trip organizer design

### PR DESCRIPTION
**Pull request by fcmatteo**
Please don't merge until tested in this branch.

### CODE REVIEW

File: src/shared_components/Cards/Trip.js [hunk 1]
```diff
@@ -148,104 +133,51 @@ class TripCard extends Component {
 
     return (
       <div>
-        {this.state.truncated ? (
```
Comment (dm0lz) [inline src/shared_components/Cards/Trip.js]: Keep this :)

File: src/shared_components/Form/SemanticLocationControl.js [hunk 2]
```diff
@@ -40,13 +43,27 @@ export default class SemanticLocationControl extends Component {
...
+      isOpen: false,
+    });
+  };
+
   render() {
-    const { inputProps, inputStyles, onlyCities } = this.props;
+    const { inputProps, inputStyles, onlyCities, useStyledInput } = this.props;
+    const { isOpen } = this.state;
 
     return (
```
Comment (dm0lz) [inline src/shared_components/Form/SemanticLocationControl.js]: This used in many different places. Please make sure it does not impact other places in the app.

File: src/shared_components/Cards/Trip.js [hunk 3]
```diff
@@ -117,25 +117,10 @@ const duration = minutes => {
 };
 
 class TripCard extends Component {
-  constructor(props) {
```
Comment (dm0lz) [inline src/shared_components/Cards/Trip.js]: keep this.

File: src/shared_components/Form/SemanticLocationControl.js (hunk 2 above)
Comment (fcmatteo) [reply src/shared_components/Form/SemanticLocationControl.js]: It seems to work fine in all the places that I tried it.

**Pull request merged**